src/screens/ENSConfirmRegisterSheet.js

```diff
@@ -28,6 +47,75 @@ export default function ENSConfirmRegisterSheet() {
     rentPrice: registrationData?.rentPrice,
   });
 
+  useEffect(() => {
+    const callbackGetRentPrice = async () => {
+      const rentPrice = await getRentPrice(name, secsInYear);
+      setRentPrice(addBuffer(rentPrice.toString(), 1.1));
+    };
+    callbackGetRentPrice();
```

Comment:
Can we not get rent price from `registrationCostsData`?

Reply:
true!